src/libs/address.js

```diff
@@ -1,11 +1,16 @@
 import IdunnPoi from '../adapters/poi/idunn_poi';
 
 /**
- * Format an address given an address object (name, city, country, label, and other regions)
+ * Filter an address and return an array with the relevant items
  * @param {*} address - an address object
  */
-export function format(address) {
-  if (!address) {return '';}
+export function toArray(address) {
+
+  if (!address) {return [];}
+
+  const cityAndPostcode = address.postcode && address.city
+    ? address.postcode + ' ' + address.city
+    : address.city;
 
   if (!address.street) {
     return [
```

Comment:
So for such addresses we can display 7 lines if everything is filled and different? This seems a bit much. I wonder if some filtering decision should be added…

Reply:
Indeed, let's @amatissart as he wrote this list

Reply:
so: in practice it never happens, and we can keep it like that for now, until a more sophisticated server-side formatting is introduced